network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -750,20 +743,20 @@ def get_banner(self):
         ),
         MultiFieldPanel(
           [
-            ImageChooserPanel('spotlight_image'),
-            FieldPanel('spotlight_headline'),
-            InlinePanel('spotlight_posts', label='Posts', min_num=3, max_num=3),
+            FieldPanel('cause_statement'),
+            FieldPanel('cause_statement_link_text'),
```

Comment:
Could we add factories for the cause statement link text and page chooser?

Reply:
definitely!